docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-engineering-guidelines.md

```diff
@@ -37,14 +41,44 @@ At the development level, we are very strict about each piece of data requested
+
+### Using app token for a user-initiated action
+
+For user-initiated actions, apps should use either `ctx.vtex.storeUserAuthToken` or `ctx.vtex.adminUserAuthToken` (as opposed to `ctx.authToken`). For more information, see [Connecting to VTEX Core Commerce APIs](https://developers.vtex.com/vtex-developer-docs/docs/how-to-connect-with-vtex-core-commerce-apis-using-vtex-io#steps).
+
+### Declaring broad policies
+
+Outbound access policies (`outbound-access`) to VTEX resources should follow the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege). In short, you only give access to what is really needed. For instance: the `path` should not be configured as `*`. See an example [here](https://github.com/vtex-apps/store-graphql/blob/684dcbbbd6e9cdbd121afd7802200856cb952d2b/manifest.json#L107-L112) on how it should be done.
+
+### Avoid exposing private information through public routes
+
+Public routes do not require authentication when called from the Frontend, so they should not expose information obtained from calling private APIs, e.g.: exposing a list of orders to the Frontend. A common mistake is leaving routes which are meant to only be consumed from the Admin unrestricted. Also, GraphQL APIs are public by default, so the schema should not have queries exposing private information either.
+
+Access to REST APIs should be restricted using [policies](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-policies#resource-based-policies). Access to GraphQL APIs should be restricted using directives, either [@auth](https://github.com/vtex/node-vtex-api/blob/08ea11d380997f5abf02455487b342caa74b2001/src/service/worker/runtime/graphql/schema/schemaDirectives/Auth.ts#L66-L75) or a custom-made one.
+
+### Isolate data between tenants
+
+Single-tenant external systems should be used carefully. Data from one tenant should always be isolated from data from another tenant to prevent leakage and unwanted access.
+
+### Persisting PII within VTEX
+
+If using Master Data or VBase to store Personal Indentifiable Information (PII), a mechanism should be in place to ensure compliance with the [Right to be Forgotten](https://www.gdpreu.org/right-to-be-forgotten/) or other similar data-protection practices.
+
+### Sending PII to external service
+
+We should ensure that all PII being shared externally is strictly necessary for the app to work. If you have any questions, reach out our team by [opening a ticket](https://help-tickets.vtex.com/smartlink/sso/login/zendesk).
+
 ## Coding standards
```

Comment:
I think this section should be removed. We haven't been enforcing this because there's no practical implication in not following it.